test/test_basic.py

```diff
@@ -126,6 +126,16 @@ def test_volume(request):
     assert v1 not in model.groups['mat:fuel']
 
 
+def test_surface(request):
```

Comment:
Should we also test the setters?

Reply:
Good call -- I'll add a check for the setters